src/test/java/com/wikia/webdriver/common/logging/Log.java

```diff
@@ -272,13 +284,55 @@ public static void stop() {
           if (url.getHost().contains("wikia")) {
             boolean isHttps = entry.getRequest().getUrl().startsWith("https");
             Log.log("VISITED URL", "Url: " + entry.getRequest().getUrl(),
-                    !Configuration.getForceHttps() || isHttps);
+                    !Configuration.getForceHttps() || isHttps
+            );
           }
         } catch (MalformedURLException e) {
           Log.log("MALFORMED URL", "Url: " + entry.getRequest().getUrl(), false);
         }
       }
     }
+
+    if (driver.getProxy() != null && Configuration.getAdsData()) {
+      Har har = driver.getProxy().getHar();
+      for (HarEntry entry : har.getLog().getEntries()) {
+        try {
+          if (entry.getRequest().getUrl().contains("adeng")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS134)